<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineWin</class>
 <widget class="QWidget" name="LineWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="create_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>create line</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 2, 6);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>edit line</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>delete line</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
